test/libyul/evmCodeTransform/after_terminating_function.yul

```diff
@@ -0,0 +1,22 @@
+{
+	let b := f(1, 2)
+	function f(a, r) -> t { revert(0, 0) }
+	b := f(3, 4)
+}
+// ====
+// stackOptimization: true
+// ----
+//     /* "":17:18   */
```

Comment:
Not related to the PR but I just noticed that you have these comments in all evm transform tests and they seem pretty useless. `AsmPrinter` now allows disabling them (#11944).